src/canvas/tileLayer.js

```diff
@@ -0,0 +1,87 @@
+geo.canvas.tileLayer = function () {
```

Comment:
@manthey and I talked and once we switch to quadFeature for d3, this code may no longer be required.